server/tests/unit/services/RunService.test.js

```diff
@@ -10,37 +10,194 @@ afterAll(async done => {
+                    where: { git_hash: '826d2c2a6a9c5b9c7045d6fa7e1b575ce4fb9762' }
+                });
+                const apgExample2 = await db.ApgExample.findOne({
+                    where: { test_version_id: testVersion2.id }
+                });
+                const at2 = await db.At.findOne({
+                    where: { test_version_id: testVersion2.id },
+                    include: [ db.AtName ]
+                });
+
+                const browser = await db.Browser.findOne();
+                let browserVersion = await db.BrowserVersion.create({
+                    browser_id: browser.id,
+                    version: browserVersionNumber
+                })
+
+                let runStatus = await db.RunStatus.findOne({
+                    where: { name: 'raw' }
+                });
+
+                // We will need to remove this column!!
+                const user = await db.Users.create();
+                const testCycle = await db.TestCycle.create({
                     test_version_id: testVersion.id,
-                    key: atKey
+                    created_user_id: user.id
                 });
-                const browser = await db.Browser.findOne({
-                    where: { name: db.Browser.CHROME }
+
+                // Create
+                const data = {
+                    test_version_id: testVersion.id,
+                    apg_example_ids: [apgExample.id],
+                    at_browser_pairs: [
+                        {
+                            at_name_id: at.at_name_id,
+                            at_version: atVersionNumber,
+                            browser_id: browser.id,
+                            browser_version: browserVersionNumber
+                        }
+                    ]
+                };
+
+                const activeRuns = await RunService.configureRuns(data);
+                const keys = Object.keys(activeRuns);
+                const runId = keys[0];
+                delete activeRuns[runId].id; // Deleting id because we don't know it
+                expect(keys.length).toEqual(1);
+                expect(activeRuns[runId]).toEqual({
+                    browser_id: browser.id,
+                    browser_version: browserVersionNumber,
+                    browser_name: browser.name,
+                    at_id: at.id,
+                    at_key: at.key,
+                    at_name: at.AtName.name,
+                    at_version: atVersionNumber,
+                    apg_example_directory: apgExample.directory,
+                    apg_example_name: apgExample.name,
+                    apg_example_id: apgExample.id,
+                    run_status_id: runStatus.id,
+                    run_status: runStatus.name,
+                    test_version_id: testVersion.id,
+                    testers: []
                 });
+            });
+        });
+        it('Activate deactived run', async () => {
```

Comment:
What is this test testing... Can you name it more clearly? 

What is the other test you want to add. 

I'm confused what you are including in your naive version of this implementation.